example/cumulus-tf/variables.tf

```
@@ -194,6 +182,28 @@ variable "buckets" {
   default = {}
 }
 
+variable "cloudwatch_log_retention_periods" {
+  type = map(number)
+  description = "retention periods for the respective cloudwatch log group, these values will be used instead of default retention days"
+  default = {
+    data-migration2 = 7,
+    DiscoverGranules = 7,           # ingest module
+    granuleFilesCacheUpdater = 7,   # archive module
+    ProvisionPostgresDatabase = 7,  # cumulus-rds-tf module
+    DistributionApiEndpoints = 7,   # cumulus_distribution module
+    EcsLogs = 7,                    # cumulus_ecs_service module
+    s3-credentials-endpoint = 7,    # distribution module
+    PrivateApiLambda = 7,           # archive module
+    TeaCache = 7,                   # tea-map-cache module 
+  }
+}
+
+variable "default_log_retention_days" {
+  type = number
+  default = 30
```

Please change ALL `example` deployment variables to have different value from the default of `tf-modules/*`, so we can verify this configuration parameter works.
```suggestion
  default = 14
```